packages/ui/test/council/pages/Election.test.tsx

```diff
@@ -40,9 +40,13 @@ jest.mock('../../../src/memberships/hooks/useMemberCandidacyStats', () => ({
 const aliceMemberId = getMember('alice').id
 const bobMemberId = getMember('bob').id
 
+// Easier to test if both ID are the equal
```

Comment:
```suggestion
// NOTE in these tests, ids of candidates and members/options are used interchangeably for simplicity
```
It's a detail but I got a little confused when I saw that `castVote()` was passed candidate ids for `optionId`s. Maybe this could make it a little clearer WDYT ?